website/pages/ko/_meta.js

```diff
@@ -1,5 +1,5 @@
 import meta from '../en/_meta.js'
 
 export default {
-  ...meta,
+  ...JSON.parse(JSON.stringify(meta)),
```

Comment:
is meta not already JS object?

Reply:
it's for copy object, Nextra enhance title and since we use same `_meta` in different language (for sidebar order) we can have a bug to have another translation in the sidebar

Reply:
Could we use [structuredClone()](https://developer.mozilla.org/en-US/docs/Web/API/structuredClone) instead?